__mocks__/react-native-localize.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+const getLocales = () => [
+  {
+    countryCode: 'US',
+    languageTag: 'en-US',
+    languageCode: 'en',
+    isRTL: false,
+  },
+  {
+    countryCode: 'FR',
+    languageTag: 'fr-FR',
+    languageCode: 'fr',
+    isRTL: false,
+  },
+];
+
+const findBestAvailableLanguage = () => ({
+  languageTag: 'en-GB',
+  isRTL: false,
+});
+
+const getNumberFormatSettings = () => ({
+  decimalSeparator: '.',
+  groupingSeparator: ',',
+});
+
+const getCalendar = () => 'gregorian';
+const getCountry = () => 'GB';
+const getCurrencies = () => ['GBP', 'EUR'];
+const getTemperatureUnit = () => 'celsius';
+const getTimeZone = () => 'Europe/London';
```

Comment:
so all users will get dates in London time zone and not their own? 🤔

Reply:
Well, that's the time standard!

Reply:
As much as I agree with Vitor on this one, this is merely a mock for tests (as react-native-localize is a native module), so it won't override user settings :)

Reply:
oh my bad sorry 😅

Reply:
No worries! Always better to double check, in my opinion :)